docs/subscription-plans.rst

```diff
@@ -63,6 +63,47 @@ optimized**.
   times the storage capacity.
 
 
+.. _subscription-plans-contracts:
+
+The CrateDB Cloud Contract
+==========================
+
+The **CrateDB Cloud Contract** allows you to pay for a full year's worth of the
+service of your choice in advance. You purchase a certain number of DTUs for
+one of the subscription plans mentioned above, and pay them up front for the
+full year. Depending on the specifics of the contract chosen, it may be
+possible to negotiate a discount based on the up front payment. For more
+information, contact our `Sales team`_.
+
+The process depends on whether you sign up via Azure or via AWS, as described
+below:
+
+
+Azure
+-----
+
+To sign a CrateDB Cloud Contract via Microsoft Azure, follow the `initial steps
+for signup`_ while selecting the CrateDB Cloud Contract as your subscription
+plan. This will automatically alert our Sales team, who will get in touch with
+you to configure the specifics of your contract according to your needs.
+
+
+AWS
+---
+
+To sign a CrateDB Cloud Contract via AWS, simply follow the `link`_ to the
```

Comment:
as a general rule of thumb, it is better to phrase hyperlinks so that the link text itself functions as a noun phrase

so in this case, I would reword this like so:

> To sign a CrateDB Cloud Contract via AWS, visit the [contract page](http://example.com/ on the AWS Marketplace.

a good way to think about this is to consider the fact that some screenreaders (web browsers designed for blind and visually impaired visitors) construct a separate list of all links in the document as a sort of erzatz navigation menu

to imagine what this is like, take every link and add the link text (by itself) to a bullet point list. then consider whether that those links will make sense devoid of their surrounding context

so, for example, a portion of the link menu for this document might read:

* Azure Marketplace
* AWS Marketplace
* Sales team (mailto)
* initial steps for signup
* link
* Azure offer page
* AWS offer page

in this instance, "link" does not indicate to the visitor where the link will take them, so it is a good candidate for being rephrased

I will add a note about this to our style guide

Reply:
Aha, I did not know that! That is an excellent point